### ISSUE TITLE
[Feature request] Ability to offset start position

### ISSUE THREAD

**Issue by archiewood** [abosch19/svelte-fast-marquee#3]
Sometimes you want to put multiple rows of marqees.

I would like to be able to stagger them:

Currently:
<img width="1282" alt="CleanShot 2023-09-05 at 10 53 38@2x" src="https://github.com/abosch19/svelte-fast-marquee/assets/58074498/47829ee4-ffd9-40aa-b8a9-d409a29c8b92">

Desired: 
<img width="1262" alt="CleanShot 2023-09-05 at 10 54 32@2x" src="https://github.com/abosch19/svelte-fast-marquee/assets/58074498/43fa7e87-c23b-4b57-a536-1ce48c9d9ce6">

I am open to contributing this

**Comment by archiewood** [comment]
consumer workaround is to put inside div and add negative margin

```svelte

<div>
  <Marquee/>
</div>
<div class=offset>
  <Marquee/>
</div>

<style>
.offset {
  margin-left: -30px;
}
</style>

```

**Comment by abosch19** [comment]
@archiewood this design seems awesome! I think the best way to solve this approach is using class and css as your example. Component has a `class` prop but I saw it was not in component types.

Do you find useful this solution?

```svelte
<Marquee/>
<Marquee class="offset"/>

<style>
:global(.offset) {
  margin-left: -30px;
}
</style>
```

**Comment by archiewood** [comment]
This looks cleaner, thanks!

**Comment by abosch19** [comment]
Closing issue. Typings solved in version 0.5.3.
